BullsAndCows/GameRunner.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BullsAndCows
+{
+    public class GameRunner
+    {
+        public void Run()
+        {
+            SecretGenerator secretGenerator = new SecretGenerator();
+            var game = new BullsAndCowsGame(secretGenerator);
+            var console = new ConsoleInteraction();
+            var correctAnswer = "4A0B";
+
+            while (game.CanContinue)
```

Comment:
Can be refactored. Now using 2 flags to control the while loop.

Reply:
Good suggestion. Resolved